tools/meteor-services/deploy.js

```diff
@@ -368,7 +370,8 @@ var bundleAndDeploy = function (options) {
   var preflight = authedRpc({
     site: site,
     preflight: true,
-    promptIfAuthFails: promptIfAuthFails
+    promptIfAuthFails: promptIfAuthFails,
+    qs: options.rawOptions
```

Comment:
I think perhaps we don't actually have to pass `qs` here -- this request seems to just do the first half of the deploy anyway (i.e. the info part). I'm not entirely sure I understand the purpose of this, TBH.